src/Bridge/Doctrine/Orm/Extension/PaginationExtension.php

```diff
@@ -92,8 +96,18 @@ public function applyToCollection(QueryBuilder $queryBuilder, QueryNameGenerator
             throw new InvalidArgumentException('Item per page parameter should not be less than or equal to 0');
         }
 
+        $firstResult = ($request->query->get($this->pageParameterName, 1) - 1) * $itemsPerPage;
+        if ($request->attributes->get('_graphql')) {
+            $collectionArgs = $request->attributes->get('_graphql_collections_args', []);
+            if (isset($collectionArgs[$resourceClass]['after'])) {
+                $after = base64_decode($collectionArgs[$resourceClass]['after'], true);
```

Comment:
`\base64_decode` as below? Don't remember the policy on this but we need it to be consistent.

Reply:
We should use `\` everywhere, it's faster. IIRC, there is a PHP CS Fixer rule for that.